service/create/assets.go

```diff
@@ -21,3 +24,31 @@ func (s *Service) encodeTLSAssets(assets certificatetpr.AssetsBundle, svc *kms.K
 
 	return compTLS, nil
 }
+
+func (s *Service) encodeKeyAssets(assets map[randomkeytpr.Key][]byte, svc *kms.KMS, kmsKeyArn string) (*randomkeytpr.CompactRandomKeyAssets, error) {
+
+	encryptionKey, ok := assets[randomkeytpr.EncryptionKey]
```

Comment:
This is not generic. When another key was added this has to be adapted as well. Is this on purpose?

Reply:
We should have this key in the map, So I was thinking to validate it later on, like we do for certificates:

WDYT?

Reply:
I think we should move forward now because we will change nearly everything when we start  using Cloudformation.

Reply:
Checking we have the right keys makes sense to me.